acceptance/tests/resource/mount/destroy.rb

```diff
@@ -21,31 +24,28 @@
   end
 
   #------- SETUP -------#
-  step "(setup) backup fstab file"
-  on(agent, "cp #{fstab} /tmp/fstab", :acceptable_exit_codes => [0,1])
-
-  step "(setup) create disk image"
-  on(agent, "dd if=/dev/zero of=/tmp/#{name} count=10240", :acceptable_exit_codes => [0,1])
-  on(agent, "yes | mkfs -t ext3 -q /tmp/#{name}")
+  step "(setup) backup #{fs_file} file"
+  on(agent, "cp #{fs_file} /tmp/fs_backup_file", :acceptable_exit_codes => [0,1])
 
   step "(setup) create mount point"
   on(agent, "mkdir /#{name}", :acceptable_exit_codes => [0,1])
 
-  step "(setup) add entry to filesystem table"
-  on(agent, "echo '/tmp/#{name}  /#{name}  ext3  loop  0  0' >> #{fstab}")
+  step "(setup) create new filesystem to be mounted"
+  create_filesystem(agent, name)
+
+  if agent['platform'] !~ /aix/
+    step "(setup) add entry to filesystem table"
+    add_entry_to_filesystem_table(agent, name)
```

Comment:
^ Actually, yes - not having that method append the new filesystem to fstab everywhere is dumb. I'll update that and get rid of this bit.